scripts/manage-ruleset.sh

```
@@ -0,0 +1,295 @@
+    gh api -X POST "repos/$REPO/rulesets" --input /tmp/new-ruleset.json
+    
+    if [ $? -eq 0 ]; then
+        echo -e "${GREEN}✅ Ruleset created successfully!${NC}"
+    else
+        echo -e "${RED}❌ Failed to create ruleset${NC}"
+        return 1
+    fi
+}
+
+function update_ruleset() {
+    local ruleset_id=$(get_ruleset_id)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)